articles/as_long_as.md

```diff
@@ -0,0 +1,29 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie
+issue: 96
+title: “As long as you put your Feet Under my Table” - If the Project Manager Leads Differently than I Would 
+---
+# {{page.title}}
+
+“As long as you put your Feet Under my Table” - If the Project Manager Leads Differently than I Would 
+ 
+After 1.5 years of solely working with Zühlke people, a colleague transferred to a project team with people from several service providers. After a few days he complained that the way of working and the approach in his new project was so completely different compared to what he was used to when working with Zühlke teams. He was afraid that this would not become a successful project following this approach.
+
+Another colleague switched from several Zühlke-led projects to another project in which he was the only Zühlke employee. After one month where procedures within the project seemed to have changed arbitrarily on a daily basis and with inexistent internal communication, he was frustrated and wanted to leave the project.
+
+I joined and supported this project team during the transitional period for 2 months. After a few days only it became already clear to me that I would not be able to work efficiently in this project because leadership and communication were so different from how I would have set up and managed the project.
+
+These are three real examples from my close environment and three times the question is: Is it bad just because it is different and how do I best deal with this “differences" ?
+ 
+To use the words from Henry Ford I see three ways of how to act: “Love it, leave it, or change it”
+ 
+The most gallant way to deal with the situation is to accept it. But not only accept but try to understand and then support the way of working, the maybe different mindset and change the own behavior in a way it will be aligned to the others. The benefit of this approach is that it will cause little friction within the project. Yet the most important point from my point of view is, that you are able to try different ways of working, new approaches and have the possibility to learn and increase your personal level of growth as you are on a new path you would maybe never chose by yourself and you have the possibility to gain further experiences. Key point from my point of view is to remain true to yourself and to show your integrity at all times.
```

Comment:
"most gallant way" could be "most elegant way" which sounds better... to me at least ;-)